pkg/purser_plugin/costing.go

```diff
@@ -222,3 +223,37 @@ func GetAllNodesCost() {
 	printNodeDetails(nodes)
 	//fmt.Println(nodes)
 }
+
+/*
+GetGroupCost returns Cost (total, cpu, memory and storage) for a Group
+*/
+func GetGroupCost(group *crd.Group) Cost {
+	cpuCostPerCPUPerHour, memCostPerGBPerHour, storageCostPerGBPerHour := GetUserCosts()
+	cost := Cost{}
+	cost.cpuCost = 0.0
```

Comment:
Initialising struct fields is not required since in Go the fields are by default set to their zero values.